dev-qt/qtwebengine/qtwebengine-5.11.1.ebuild

```diff
@@ -79,6 +79,8 @@ DEPEND="${RDEPEND}
 	pax_kernel? ( sys-apps/elfix )
 "
 
+PATCHES+=( "${FILESDIR}/${P}-libxml2-disable-catalogs.patch" )
```

Comment:
can you reference a bug# either here or in the commit msg?

Reply:
not sure why github shows this diff as "outdated"...

Reply:
ping on this, the rest LGTM

Reply:
thanks, done!